terraform/stacks/tooling/outputs.tf

```diff
@@ -317,23 +316,6 @@ output "master_bosh_secret_access_key_curr" {
   value = "${module.master_bosh_user.secret_access_key_curr}"
 }
 
-/* tooling bosh user */
-output "tooling_bosh_username" {
-  value = "${module.tooling_bosh_user.username}"
-}
-output "tooling_bosh_access_key_id_prev" {
-  value = "${module.tooling_bosh_user.access_key_id_prev}"
-}
-output "tooling_bosh_secret_access_key_prev" {
-  value = "${module.tooling_bosh_user.secret_access_key_prev}"
-}
-output "tooling_bosh_access_key_id_curr" {
-  value = "${module.tooling_bosh_user.access_key_id_curr}"
-}
-output "tooling_bosh_secret_access_key_curr" {
-  value = "${module.tooling_bosh_user.secret_access_key_curr}"
-}
-
```

Comment:
So `cg-deploy-bosh` still uses this user for the following stanza. Looks like we'll need to bring it back. 😕

```yaml
- name: semver-tooling-version
  type: semver
  source:
    driver: s3
    bucket: {{semver-bucket}}
    key: {{semver-tooling-key}}
    access_key_id: {{semver-access-key-id-tooling}}
    secret_access_key: {{semver-secret-access-key-tooling}}
    region_name: {{aws-region}}
```